.github/workflows/scripts/coverage.sh

```diff
@@ -27,7 +27,7 @@ BINARIES=""
 for file in \
   $( \
     RUSTFLAGS="-Zinstrument-coverage" \
-      cargo +nightly test --tests --all --all-features --no-run --message-format=json \
+      cargo +nightly test --tests --all --all-features --no-run --block-format=json \
```

Comment:
Is this correct?

Reply:
No, I still have to go through all files and check because I just did some global raplaces